app/routes/dashboard.php

```diff
@@ -66,8 +66,12 @@
     Route::group(['prefix' => 'notifications'], function () {
         Route::get('/', [
             'as'   => 'dashboard.notifications',
-            'uses' => 'DashboardController@showNotifications',
+            'uses' => 'NotificationController@showSlack',
         ]);
+        Route::get('slack', 'NotificationController@showSlack');
```

Comment:
I don't know how to redirect / to /slack. So actually, we have two routes to go at the same place...